pkg/server/container_create_windows.go

```diff
@@ -475,6 +475,7 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 		}
 
 		setOCINamespaces(&g, securityContext.GetNamespaceOptions(), sandboxPid)
+		c.addOCIWindowsDevices(&g, config.Devices)
```

Comment:
Please add a comment explaining why we're calling a function that says `WindowsDevices` in the Linux section. :)